### ISSUE TITLE
Add SRW papers to srw_papers.csv

### ISSUE THREAD

**Issue by hao-fang** [acl-org/acl-2020-virtual-conference#73]
**Number of Volunteers**: 1

Follow-up on #8, we will need to add SRW papers to the `srw_papers.csv` file.

- [x] Get information for SRW chairs (after #74 is done)

**Comment by penny9287** [comment]
I can work on this  issue.

**Comment by ardakdemir** [comment]
@hao-fang  Where should I get the relevant files for SRW papers to add them to papers.csv?

**Comment by hao-fang** [comment]
@ardakdemir Here are the papers. Please let me know if you need any additional information.

[ACL SRW 2020 Live QA Slots (for website).xlsx](https://github.com/acl-org/acl-2020-virtual-conference/files/4801679/ACL.SRW.2020.Live.QA.Slots.for.website.xlsx)

**Comment by ardakdemir** [comment]
Thanks!

**Comment by hao-fang** [comment]
Closing this, but I open a follow-up issue #172 .
